derp/derp.go

```diff
@@ -2,7 +2,8 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// Package derp implements DERP, the Detour Encrypted Routing Protocol.
+// Package derp implements DERP, the Designated Encrypted Relay
```

Comment:
I think the meeting ended on keeping it as either "Detour Encrypted Routing Protocol" or maybe making Detour be "Detoured". I forget.